app/assets/stylesheets/comp-modals.scss

```diff
@@ -31,4 +31,150 @@
 .mfp-close-btn-in .mfp-close:hover {
 	color: #FFF;
 	border-radius: 0;
-}
\ No newline at end of file
+}
+.modal-attachments {
```

Comment:
Maybe modal-slim? The default modal has a different look (with more padding), so I didn't want to break styles used elsewhere.